src/power_grid_model_io/data_stores/excel_file_store.py

```diff
@@ -28,9 +41,16 @@ class ExcelFileStore(BaseDataStore[TabularData]):
 
     _unnamed_pattern: re.Pattern = re.compile(r"Unnamed: \d+_level_\d+")
 
-    def __init__(self, file_path: Optional[Path] = None, **extra_paths: Path):
+    def __init__(
+        self,
+        file_path: Optional[Path] = None,
+        language: str = "en",
+        terms_changed: Optional[dict] = None,
+        **extra_paths: Path,
+    ):
         super().__init__()
-
+        if not isinstance(language, str):
```

Comment:
? That's the users' responsibility tho. And we have mypy on our end

Or is this for compatibility? In that case we can also just bump mjnor version

Reply:
Perhaps something for discussion next Tuesday.

Reply:
Let's discuss during standup instead, then with a bit of luck we can decide on the path going forward and merge this PR today or tomorrow morning

Reply:
Included in the newest commmit.